sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c

```diff
@@ -243,7 +259,7 @@ void gnrc_sixlowpan_frag_send(gnrc_pktsnip_t *pkt, void *ctx, unsigned page)
     }
 #endif
 
-    /* Check weater to send the first or an Nth fragment */
+    /* Check whether to send the first or an Nth fragment */
```

Comment:
Please put this spelling fix in a separate commit.

Reply:
done